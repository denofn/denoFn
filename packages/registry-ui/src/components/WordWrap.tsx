import React from "react";
import { css } from "otion";

export function WordWrap({ children }: React.PropsWithChildren<{}>) {
  return (
    <span
      className={css({
        wordBreak: "break-all",
      })}
    >
      {children}
    </span>
  );
}
