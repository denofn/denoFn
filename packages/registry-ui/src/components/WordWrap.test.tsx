import { render, cleanup } from "@testing-library/react";
import React from "react";

import { WordWrap } from "./WordWrap";

afterEach(cleanup);

it("renders with the correct styles", () => {
  const { asFragment } = render(<WordWrap />);
  expect(asFragment()).toMatchSnapshot();
});
