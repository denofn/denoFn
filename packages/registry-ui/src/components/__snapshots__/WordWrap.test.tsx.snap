// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the correct styles 1`] = `
<DocumentFragment>
  <span
    class="_1m5hyg0"
  />
</DocumentFragment>
`;
